build/icon-builder/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM php:7
```

Comment:
Looks like it belongs in https://github.com/nextcloud/docker-ci? Building this image in every run is not very efficient.

Reply:
This might be a good idea. I would do so, iff the building of the images is a wished feature for the build process. Currently, the most time is spent when the images have to be built (thanks to the cache, this should be done only weekly).

If there is a better (existing?) alternative (circleCI?), I'd avoid the trouble to sync both repositories.